csv/parse.ts

```diff
@@ -286,7 +287,8 @@ class Parser {
   }
 }
 
-export interface ParseOptions extends ReadOptions {
+export interface ParseOptions<Columns extends string = string>
+  extends ReadOptions {
   /**
```

Comment:
Note: In order to achieve the following, I had to make the `ParseOptions` type generic and let TypeScript guess the concrete values for `columns`.

```ts
parse("", { columns: ["aaa", "bbb"] }); //=> Record<"aaa" | "bbb", string>[]
```